### PR TITLE
VACMS-6602: Set login time from last to time.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Relates to #6602

## Testing done
Visual / behat

## Screenshots
image

## QA steps
 - [ ] As an administrator, go to `admin/people` and visually verify last column reads `LAST ACCESS (USA NYC TIME)`
 - [ ] Visually verify last column login times are in `30-Sep-21 7:37pm` format
 - [ ] Download csv from link at bottom of page, and visually verify time format for last login is `30-Sep-21 7:37pm`

### CODE REVIEW

File: config/sync/views.view.user_admin_people.yml [hunk 1]
```diff
@@ -634,7 +635,7 @@ display:
           relationship: none
           group_type: group
           admin_label: ''
-          label: 'Last access'
+          label: 'Last access (USA NYC time)'
```
Comment (swirtSJW) [inline config/sync/views.view.user_admin_people.yml:638]: Are we sure of it being for NY?  Doesn't it use the users local time?

Comment (swirtSJW) [reply config/sync/views.view.user_admin_people.yml:638]: Confirmed.  It does NOT use NY.   It uses time relative to logged in user viewing the View
image

Pacific time on the left, NY time on the right.

**Review by swirtSJW — CHANGES_REQUESTED** [review]
We should take out the reference to the time zone on the header.

**Review by kevwalsh — CHANGES_REQUESTED** [review]
This is great and has unblocked some of the user audit work. 

A few changes i'd like to request

- [x]  Revert to "time ago" dates for Last Access on the /admin/people page 
- [x]  In the CSV output
  - [x] switch to non-relative dates for both the Member Since AND the Last Access fields
  - [ ] Use the date format "October 5, 2021, 5:21pm ET" on the CSV
- [ ] What Steve said (revert to the old column field labels, in both the CSV and the view screen)
- [ ] Stretch: on the CSV output: add a new column with an "Edit link" column label, and output a raw link, eg, https://prod.cms.va.gov/user/UID/edit

**Comment by ethanteague** [comment]
Updated @swirtSJW @kevwalsh :+1:

**Review by kevwalsh — APPROVED** [review]
Looks good to me. Sent to Chris and Stef and Jane for review.

**Review by swirtSJW — APPROVED** [review]
Looks great.  Thanks Ethan.
image


image

**Pull request merged**